media-sound/audacity/audacity-2.0.5.ebuild

```diff
@@ -1,101 +0,0 @@
-# Copyright 1999-2015 Gentoo Foundation
-# Distributed under the terms of the GNU General Public License v2
-
-EAPI=5
-
-inherit eutils wxwidgets autotools versionator
-
-MY_PV=$(replace_version_separator 3 -)
-MY_P="${PN}-src-${MY_PV}"
-MY_T="${PN}-minsrc-${MY_PV}"
-DESCRIPTION="Free crossplatform audio editor"
-HOMEPAGE="http://web.audacityteam.org/"
-SRC_URI="mirror://gentoo/${MY_T}.tar.xz"
-
-LICENSE="GPL-2"
-SLOT="0"
-KEYWORDS="amd64 ~mips ppc ppc64 x86"
```

Comment:
hmm it seems you're killing last ppc stable

dont remove any ebuild in this pr, just update